packages/sui-react-web-vitals/src/index.js

```diff
@@ -58,7 +60,60 @@ export default function WebVitalsReporter({
       return deviceType || browser?.deviceType
     }
 
-    const handleReport = ({name, value}) => {
+    const handleAllChanges = ({attribution, name, value}) => {
+      const pathname = getPathname()
+      const routeid = getRouteid()
+      const type = getDeviceType()
+      const isExcluded =
+        !pathname || (Array.isArray(pathnames) && !pathnames.includes(pathname))
+
+      if (isExcluded) {
+        return
+      }
+
+      const amount = name === METRICS.CLS ? value * 1000 : value
+
+      logger.log({
```

Comment:
## What do you think about defining a threshold to avoid sending good INP values to 🍄?

### Define the threshold following the Core Web Vitals values

```js
const CWV_THRESHOLDS = {
  LCP: 2500,
  INP: 200
}
```

```suggestion
      const isOverThreshold = value > CWV_THRESHOLDS[name]
      isOverThreshold && logger.log({
```

Reply:
good point, indeed. Will refactor accordingly ASAP!

Reply:
Perhaps thresholds configuration could be a prop ? WDYT ?

Reply:
good point too. We could have Google's thresholds as default values and add a `prop` so they can be overridden... IMHO it can be useful for further perf fine-tuning.